### ISSUE TITLE
[Feature Request] Suggustion 

### ISSUE THREAD

**Issue by deusyss** [AntumDeluge/debreate#13]
Hi,

Once again, thanks for this good software.

I think it could be a good idea to add a command (build or create) to directly build a deb package from a dbp file. It so could be use in continuous integration directly

**Comment by AntumDeluge** [comment]
Thank you very much @deusyss. I think this is a very good idea & will add it to my TODO list.

**Comment by deusyss** [comment]
Hi, thanks for your quick answer (and sorry from being late to answer myself). Happy to see it in TODO list ^^

**Comment by damody** [comment]
I need this function too, thanks.

**Comment by AntumDeluge** [comment]
I apologize that I haven't integrated this yet. I have been busy with another project. And it doesn't help that my Linux/Debian computer broke down. So I've got to set up another one before I can really do any more work on Debreate.

**Comment by deusyss** [comment]
Hi,
Do you have any dev docs? like UML diagrams, ...

**Comment by AntumDeluge** [comment]
@deusyss: No. But I will look into it & see if it's something I want to create.

**Comment by witalij-s** [comment]
Hi, today I was using debreate again. Great job on the application! I would love a headless version to where i can just put `debreate build <name>.dbp` and it would automatically count the minor release version number and create a .deb package.

Keep up the great work :+1:

**Comment by AntumDeluge** [comment]
Thank you for the suggestion @witalij-s. I hope that I will be able to get back to working on Debreate soon. I have to get a new Linux/Debian system set up as my old one crashed & am currently using Windows.

**Comment by benelgiac** [comment]
Hi, debreate is very useful. Just +1 on this feature request, it would really be great to have that for CI! Wish you good luck with your Linux box!

**Comment by AntumDeluge** [comment]
Thank you @benelgiac. 😃

**Comment by AntumDeluge** [comment]
@benelgiac : That honestly makes me feel really good that you went to lengths to create additional functionality for Debreate. I definitely want to look over what you have done.

I want to apologize to everyone for not having worked on Debreate. As I mentioned before of the issue of getting a Linux box set up, in addition to [another project](https://github.com/arianne/stendhal) I am part of that recently needed a major overhaul, & simply life have all been in the way.

I have been struggling with figuring out a better implementation for Debreate's .dbp file format. I am going to create a new thread in hopes of getting some suggestions. I'll post a link here.

**Comment by saddy001** [comment]
Thanks for this software. I came here in need for this feature too.

**Comment by benelgiac** [comment]
Hi saddy001

if you really need it, you can try my fork (on the 0.7 branch). As I told,
it's not superclean code but I've been using it in the last months and it
does the job for me.

Cheers

**Comment by saddy001** [comment]
@benelgiac Yes thanks for that, but as I mentioned, I need the launcher icons. Your docs say they aren't created. Is it up to date?

**Comment by benelgiac** [comment]
You know, I really don't remember right now, it was some time ago, sorry...
But I will try and check to see what I meant exactly.

Il sab 25 gen 2020, 10:00 saddy001 <notifications@github.com> ha scritto:
